tests/backuprestore/backup_restore_suite_test.go

```diff
@@ -69,7 +72,9 @@ func TestE2E(t *testing.T) {
 // This setup will run once for the entire test suite
 var _ = BeforeSuite(func() {
 	project = nil
-
+	if envLabelFilter := os.Getenv("TEST_LABEL_FILTER"); envLabelFilter != "" {
+		isTestRelatedS3 = !strings.Contains(envLabelFilter, "local")
```

Comment:
```suggestion
		isTestRelatedS3 = !strings.Contains(envLabelFilter, "installation")
```
We can skip s3 test at test level